recipes/thucyd/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "thucyd" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "e3c0c736746b0ac26b93fa8d2fcd2d90394adb080d60a84144aabf9f83325381"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy >=1.14
```

Comment:
I don't think you need `nump`y here:
```suggestion
```

Reply:
Hi Chris, why do you say this? In order to run: numpy > 1.14 I'd need that on the host. On line 13 of [my source](https://gitlab.com/thucyd-dev/thucyd/blob/master/src/thucyd/eigen/impl.py) I import numpy. 

I suppose I'm missing something about the conda-forge build. Please let me know.

Reply:
The `host` dependencies are those required to install the package (i.e. run your `setup.py` script). If you don't import numpy there (or use it's headers for compiling an extension) then you don't need it in `host`.

Reply:
Oh, I see. That's clear. I'll remove -- thanks.

Reply:
committed and built.